assets/src/scripts/outputs-viewer/hooks/use-file.js

```diff
@@ -44,7 +33,7 @@ function useFile(file) {
             responseType: "blob",
           })
           .then((response) => response.data)
-          .then((blob) => convertBlobToBase64(blob))
+          .then(() => file.url)
```

Comment:
Why are we switching to the image URL from blobs?

Reply:
I've updated the description to cover this. Previously React was storing the base64 blob, but the server is returning the correct caching headers, so the image isn't reloaded by the SPA for the same URL. That means there's no point React storing the data, when the URL will work just fine.

This now confirms that the image is available before we show the image on the client-side.